src/main/scala/core/package.scala

```diff
@@ -0,0 +1,79 @@
+import akka.actor.{Actor, ActorRef, Props}
+import core.database.DatabaseActor
+import core.scheduler.{Basched, BaschedRequest}
+import core.utils.Configuration
+import core.webserver.WebServerActor
+
+package object core {
+
+  case class PropsWithName(props: Props, actorName: String)
+
+  val TABLE_NAME_TASKS = "tasks"
+  val TABLE_NAME_RECORDS = "records"
+  val TABLE_NAME_PROJECTS = "projects"
+  val TABLE_NAME_ACTIVE_TASK = "active_task"
+
+  val PRIORITY = Map(
+    "im" -> 0,
+    "hi" -> 1,
+    "re" -> 2
+  )
+
+  val STATUS = Map(
+    "READY" -> 0,
+    "WINDOW_FINISHED" -> 1,
+    "ON_HOLD_WINDOW_FINISHED" -> 2,
+    "ON_HOLD_READY" -> 3,
+    "FINISHED" -> 4
+  )
+
+  val NUM_OF_PMDRS_PER_PRIORITY = Map(
+    PRIORITY("im") -> 0,
+    PRIORITY("hi") -> 8,
+    PRIORITY("re") -> 4
+  )
+
+  // 25m in ms
+  val POMODORO_MAX_DURATION_MS = 1500000
+
+  val ADDED = 0
+  val DUPLICATE = 1
+  val ERROR = 2
+  val UPDATED = 3
+  val SUCCESS = 4
+
+  /**
+    * Generic function to create Actor props
+    * @param newActor - function to create the Actor
+    * @return new Props object of the received Actor
+    */
+  def props(newActor: => Actor): Props = Props(newActor)
```

Comment:
I think that props should go back to each of their classes.

Reply:
will do, maybe we can do it under apply function, what do you think?

Reply:
Won't it be a bit confusing that `apply` would return `Props` as a type? 
And I'm not sure why this function bothers you.